src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,54 @@
+.calendar {
+  width: 728px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  gap: 1px;
+  padding: 10px;
+
+  &__day {
+    @extend %font-style;
+    @extend %square-style;
+    cursor: pointer;
+    transition-duration: 0.5s;
```

Comment:
Always add transition property. By default, transition property is all, and it's hard for the browser to control all properties